JSDemos/Demos/Charts/StandardBar/description.md

```diff
@@ -1 +1,38 @@
-This demo illustrates the ease with which you can create a _bar_ chart.
\ No newline at end of file
+This demo shows the standard bar series type that visualizes data as a collection of bars. Follow the steps below to create and configure the Bar chart:
+
+## Bind to Data
+
+You can bind the Bar chart to one of the following data sources: 
+
+* [Simple Array](/Documentation/Guide/Data_Binding/Specify_a_Data_Source/Local_Array/)
+* [JSON Data](/Documentation/Guide/Data_Binding/Specify_a_Data_Source/Read-Only_Data_in_JSON_Format/)
+* [OData Service](/Documentation/Guide/Data_Binding/Specify_a_Data_Source/OData/)
+* [Custom Sources](/Documentation/Guide/Data_Binding/Specify_a_Data_Source/Custom_Data_Sources/)
+
+In this demo, the Bar chart is populated with data taken from a simple JavaScript array.
+
+## Configure Series
+
+Assign one or several series to the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/) property and specify the following settings:
```

Comment:
Please review the way you have listed things below. You have subcategories, so you need to list those differently from how you have done it here. Also, use punctuation when listing in this case.

Reply:
@arminal  done, thanks.